src/Core__Object.res

```diff
@@ -1,6 +1,37 @@
 @obj external empty: unit => {..} = ""
 
-@val external is: ('a, 'b) => bool = "Object.is"
+/**
+`is` determines if two objects are identical in all contexts. Objects, arrays, records, and other non-primitives are only identical if they reference the **exact** same object in memory. Primitives like ints, floats, and strings are identical if they have the same value. `+0` and `-0` are distinct. NaN is equal to itself. See [Object.is on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/is)
+
+In most scenarios use `==` or `===` or the custom `equals` function (if provided) for the type.
+
+The `==` operator [is different in ReScript than Javascript](https://rescript-lang.org/docs/manual/latest/overview#boolean). Arrays, records and other non-primitives are equal if they have the same contents (deep equality).
+
+In ReScript, the `===` operator performs a strict equality check, like Javascript, and is similar but not identical to `is`
```

Comment:
Describing other functions and operators seem outside the scope of reference documentation. It will quickly become hard to maintain the documentation if it gets scattered across other definitions.

Reply:
Ok thanks for the feedback.